index.html

```diff
@@ -63,8 +63,9 @@
                 <nav class="primary-nav">
                   <ul class="primary-nav--level-1">
                     <li><a href="#"><span>Home</span></a></li>
-                    <li aria-haspopup="true">
+                    <li class="has-children">
                       <a href="#"><span>Products</span></a>
+                      <button class="primary-nav__button-toggle" aria-expanded="false">Toggle sub-navigation</button>
```

Comment:
we should add a `aria-controls` in here to target the ID of the triggered list

Reply:
Just pushed a fix